### DISCUSSION TITLE
Solver conditioning with L**2

### DISCUSSION THREAD

**Discussion by veera-visw** [pybamm-team/PyBaMM#3918] [Q&A] [answered]
In the basic_DFN model implementation, I can see that the potential equations are conditioned with L**2 .

![image](https://github.com/pybamm-team/PyBaMM/assets/157810026/44cdda89-bf5b-4fec-b2e9-cb0b8fe5b471)

I have tried with and without this conditioning and I see the results dont vary too much. I understand the conditioning is to help thewith speedup, but is it mandatory to keep this term in the solution? Is this dimensionally correct ?

Just wondering, thanks !

**Comment by XuboGU** [comment, marked as answer]
Hi @veera-visw , L_x should be a significant value, without it, i think, will affect the conservation of the equation. Thus, it should change the result. I suggest you present an example code for others' convenience for checking this.

**Reply by veera-visw** [reply]
I have included this term in my equations now. I dont want to reflect yet on why the absence gave similar results. But I am happy with the results I get from including the conditioning term.

Does the magnitude of this value matter? Right now, I am using the total length of the 1D domain as my conditioner.

**Reply by valentinsulzer** [reply]
The value has no effect on the results, but including it helps the numerical solver to converge more easily since it makes the overall magnitude of the equation closer to 1 (otherwise it would be very large)
